application-licensing-licensor/application-licensing-licensor-ui/src/main/resources/Licenses/Code/VelocityMacros.xml

```diff
@@ -193,34 +193,41 @@
   {{/html}}
 #end
 
-#macro (licenseButton $licenseType $cssClass $key $jsonData)
+#macro (licenseButton $licenseType $cssClass $iconName $key $jsonData)
   #set ($jsonData.licenseType = $licenseType)
-  #set ($label = $services.localization.render("licensor.${key}License.label"))
-  &lt;button class="btn btn-$cssClass licenseButton licenseButton-$escapetool.xml($key)"
-    data-button="$escapetool.xml($jsontool.serialize($jsonData))"&gt;$label&lt;/button&gt;
+  #set ($label = $escapetool.xml($services.localization.render("licensor.${key}License.label")))
+  &lt;a href="" class="action licenseButton licenseButton-$escapetool.xml($key)"
+      data-button="$escapetool.xml($jsontool.serialize($jsonData))" title="$label"&gt;
+    &lt;span class="action-icon"&gt;&lt;span class="fa fa-$iconName"&gt;&lt;/span&gt;&lt;/span&gt;##
```

Comment:
Hard-coding the usage of Font Awesome is not nice. You should at least make sure to pull the resources required by Font Awesome, but ideally you should use the icon theme.